globals.d.ts

```diff
@@ -3,3 +3,61 @@ declare let ios: boolean;
 // @ts-ignore
 declare let __DEV__: boolean;
 declare let IS_DEV: boolean;
+
+declare module 'react-native-dotenv' {
+  export const IS_TESTING: 'true' | 'false';
+  export const ENABLE_DEV_MODE: number; // TODO
+  export const SENTRY_ENDPOINT: string;
+  export const SENTRY_ENVIRONMENT:
+    | 'production'
+    | 'development'
+    | 'LocalRelease'; // TODO
```

Comment:
Where can I find a source of truth for values like this? We should aim to have all possible values tracked here.

Reply:
q for @osdnk